src/grid/styled-components.js

```diff
@@ -0,0 +1,220 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+import {styled} from '../styles/index.js';
+import {getMediaQueries} from '../helpers/responsive-helpers.js';
+import {BEHAVIOR} from './constants.js';
+
+import type {
+  SoftResponsiveT,
+  GridPropsT,
+  CellPropsT,
+  StyledGridPropsT,
+  StyledCellPropsT,
+} from './types.js';
+
+// defaults
+const GRID_COLUMNS = [4, 8, 12];
+const GRID_GUTTERS = [16, 36, 36];
+const GRID_MARGINS = [16, 36, 64];
+const GRID_MAX_WIDTH = 1280;
+
+export function Grid({
```

Comment:
Could we move out the `Grid` component into a separate file since it's not a styled component?